### PR TITLE
webclient update

### PR DESCRIPTION
**Pull request by micaela-fm**

### CODE REVIEW

File: phase-1/webclient.py
Comment (tiago-nevoa) [inline phase-1/webclient.py:1]: \n at EOF! Please refactor...

**Review by tiago-nevoa — CHANGES_REQUESTED** [review]
\n at EOF! Please refactor...

File: phase-1/webclient.py
Comment (micaela-fm) [reply phase-1/webclient.py:1]: Done

**Review by tiago-nevoa — APPROVED** [review]
lgtm! Nice work

**Pull request merged**